tileview/src/main/java/com/qozix/tileview/widgets/ZoomPanLayout.java

```diff
@@ -778,9 +794,10 @@ private boolean setupZoomAnimation( float scale ) {
       return false;
     }
 
-    public void animateZoomPan( int x, int y, float scale ) {
+    public void animateZoomPan( int x, int y, float scale, @Nullable ZoomPanListener.Origination origination ) {
```

Comment:
you're correct, https://github.com/moagrius/TileView/pull/261/files#r52179298 applies here as well